src/init.cpp

```diff
@@ -687,8 +686,8 @@ void ThreadImport(std::vector<fs::path> vImportFiles)
     } // End scope of CImportingNow
     if (gArgs.GetArg("-persistmempool", DEFAULT_PERSIST_MEMPOOL)) {
         LoadMempool();
```

Comment:
IMO we should do this instead:
```cpp
g_is_mempool_loaded = LoadMempool();
```
because, for instance, in `LoadMempool()`:
https://github.com/bitcoin/bitcoin/blob/67447ba06057b8e83f962c82491d2fe6c5211f50/src/validation.cpp#L4592-L4593